app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -62,4 +64,414 @@ class SmokeTest {
+                }.openThreeDotMenu {
+                }.openSyncedTabs {
+                    verifyNavigationToolBarHeader()
+                    verifySyncedTabsStatus()
+                }.goBack {
+                }.openThreeDotMenu {
+                }.openSettings {
+                    verifySettingsView()
+                }.goBackToBrowser {
+                }.openThreeDotMenu {
+                }.openFindInPage {
+                    verifyFindInPageSearchBarItems()
+                }.closeFindInPage {
+                }.openThreeDotMenu {
+                }.addToFirefoxHome {
+                    verifySnackBarText("Added to top sites!")
+                }.openTabDrawer {
+                }.openHomeScreen {
+                    togglePrivateBrowsingModeOnOff()
+                    verifyExistingTopSitesTabs(defaultWebPage.title)
+                    togglePrivateBrowsingModeOnOff()
+                }.openTabDrawer {
+                }.openTab(defaultWebPage.title) {
+                }.openThreeDotMenu {
+                }.openAddToHomeScreen {
+                    verifyShortcutNameField(defaultWebPage.title)
+                    clickAddShortcutButton()
+                    clickAddAutomaticallyButton()
+                    verifyShortcutIcon()
+                }.openHomeScreenShortcut(defaultWebPage.title) {
+                }.openThreeDotMenu {
+                }.bookmarkPage {
+                    verifySnackBarText("Bookmark saved!")
+                }.openThreeDotMenu {
+                }.sharePage {
+                    verifyShareAppsLayout()
+                }.closeShareDialogReturnToPage {
+                }.openThreeDotMenu {
+                }.refreshPage {
+                    verifyUrl(defaultWebPage.url.toString())
+                }.openTabDrawer {
+                    closeTabViaXButton(defaultWebPage.title)
+                }.openHomeScreen {
+                    navigationToolbar {
+                    }.enterURLAndEnterToBrowser(youtubeUrl.toUri()) {
+                        verifyBlueDot()
+                    }.openThreeDotMenu {
+                        verifyOpenInApp()
+                    }
+                }
+            }
+        }
+    }
+
+    @Test
+    fun verifyPageMainMenuItemsListInLandscapeNormalModeTest() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+        val youtubeUrl = "www.youtube.com"
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+            activityTestRule.getActivity()
+                .setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
+        }.openThreeDotMenu {
+            // verifyThreeDotMainMenuItems()
+        }.openHistory {
+            // verifyTestPageUrl(defaultWebPage.url)
+        }.goBackToBrowser {
+        }.openThreeDotMenu {
+            activityTestRule.getActivity()
+                .setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)

```

Comment:
Do you need this everytime? I think if you set that at the very beginning then the whole test runs on landscape, at least that's what I saw when trying that out

Reply:
Okay